src/pyhf/workspace.py

```diff
@@ -743,3 +743,27 @@ def sorted(cls, workspace):
         newspec['observations'].sort(key=lambda e: e['name'])
 
         return cls(newspec)
+
+
+def build(model, data, name='measurement'):
+    """
+    Build a workspace from model and data.
+
+    Args:
+        model (~pyhf.pdf.Model): A model to store into a workspace
+        model (tensor): A array holding observations to store into a workspace
+
+    Returns:
+        ~pyhf.workspace.Workspace: A new  workspace object
+
+    """
+    workspace = copy.deepcopy(model.spec)
+    workspace['version'] = '1.0.0'
+    workspace['measurements'] = [
+        {'name': name, 'config': {'poi': model.config.poi_name, 'parameters': []}}
```

Comment:
this ignores fixed parameters. @kratsg maybe we add it in a follow-up PR?

Reply:
@lukasheinrich I haven't looked at this much yet, but how much work would it take to get this all in one?

Reply:
something like this

```
param_configs = [{'name': param_name, 'inits': model.config.suggested_init()[model.config.param_set(param_name)['slice']], 'bounds': model.config.suggested_bounds()[model.config.param_set(param_name)['slice']), 'fixed': True} for param_name in model.config.parameters if model.config.suggested_fixed()[.....]]
```

Reply:
done